CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing Guidelines
+
+Thank you for considering contributing to our documentation project! We appreciate
+your efforts in helping us improve and expand our documentation. To ensure a smooth
+and consistent contribution process, please review the following guidelines before
+submitting your contribution.
+
+## Contribution Guidelines
+
+1. Familiarize yourself with the project's documentation structure, style, and content.
+You can find our documentation in the `docs/` directory.
+
+2. When creating new guides or making updates, we strongly encourage you to use the
+**Diátaxis** framework. Diátaxis provides a standardized format and set of tools for
+creating documentation, ensuring consistency across our guides. You can find more
+information about Diátaxis and how to use it in our [Diátaxis Documentation](https://diataxis.fr/).
```

Comment:
```suggestion
information about Diátaxis and how to use it in the [Diátaxis Documentation](https://diataxis.fr/).
```

Otherwise it looks like that's our project :slightly_smiling_face: